sdk/core/azure-core/src/main/java/com/azure/core/implementation/serializer/jackson/FlatteningSerializer.java

```diff
@@ -41,6 +42,8 @@
  * {'properties' : { 'name' : 'my_name' }} in the serialized payload.
  */
 class FlatteningSerializer extends StdSerializer<Object> implements ResolvableSerializer {
+    private final transient ClientLogger logger = new ClientLogger(FlatteningSerializer.class);
```

Comment:
I'd remove the transient and suppress the Spotbug issue

Reply:
Why suppress the issue?

Reply:
Yeah. The transient doesn't work here. Will suppress the issue. @conniey, local mavn run still pass but CI failed for this change